README.md

```diff
@@ -61,11 +72,56 @@ To authenticate your identity, we use a personal access token that we call `regc
+The Personal Access Token is set up to expire after a specific period of time, requiring manual renewal by CCV personnel with access to the CCV CJ Bot. 
+
+You can use the command `kubectl get secrets` to view all secrets associated with the tpl-projectportal. 
+In order to update the expiring secret, you must first delete the existing secret by using `kubectl delete secret <SECRET_NAME>` and then use the `kubectl create secret ...` command from above to generate a new replacement secret. 
+
+
+
+##### 2. Injected Env Secret
+A personal access token is also required to authenticate the email-service when it uses the Github API to grab the content from the respective content repository. Refer to the tpl-projectportal folder in the [Brown Kubernetes Engine (BKE) Deployments repository](https://github.com/brown-ccv/k8s-deploy-bke) and find the `/secrets` folder within the tpl-projectportal subfolder. The personal access token used is also the same fine-grained personal access token generated to create the Regcred secret. Please use the same secret in this case. 
+
+We use the following simple format to encrypt an env file that contains the personal access token which will be used by the email script. Refer to https://github.com/brown-ccv/k8s-deploy-bke/tree/feat-xnat-conjob-prod#handling-secrets for further instructions on how to set up SOPs encryption credentials and the commands required to encrypt and decrypt these secrets env files. 
+
+###### Kustomization.yaml
+The kustomization.yaml file is a tool for customizaing Kubernetes manifests. In our case, we use it to manage and apply the inject env secret that you have generated. 
+
+```
+resources:
+  - example-content-stale-cronjob.yml
+  - example-content-new-cronjob.yml
+  - nc-new-cronjob.yml
+  - nc-stale-cronjob.yml
+  - satx-new-cronjob.yml
+  - satx-stale-cronjob.yml
+```
+This code excerpt is from the Kustomiation.yaml in tpl-projectportal. The `resources` section specifies the Kubernetes resources that we want to be "customized" by this Kustomization file. In other words the secrets we generate from the encrypted secret.env files will be applied to the resources that we list here. 
+```
+secretGenerator:
+  - name: env
+    envs:
+      - secrets/secrets.env
+  - name: nc-env
+    envs:
+      - secrets/nc-secrets.env
+  - name: satx-env
+    envs:
+      - secrets/satx-secrets.env
+```
+This section defines secret generators, which are used to create Kubernetes secrets from enviornment variable files found in the `/secrets` folder.
+It is important to identify each secret has a specific name: `env`, `nc-env`, `satx-env` as this identifies which secret will be used by which email-service. 
+
+### Deploying the email-service
+This [script](https://github.com/brown-ccv/k8s-deploy-bke/blob/main/deploy.sh) helps to automate the deploy process of the email-service. Simply run `./deploy.sh` and the script will run through a wizard to help you deploy the right service in the right cluster. 
```

Comment:
Any changes to the email service files (updating secrets, changing the job timeline, etc.) requires the email service to be re-deployed with './deploy.sh'.